scene/gui/line_edit.cpp

```diff
@@ -352,6 +352,41 @@ void LineEdit::_delete(bool p_word, bool p_all_to_right) {
 	}
 }
 
+Point2 LineEdit::_get_right_icon_size(Ref<Texture2D> p_right_icon, bool p_get_min) const {
+	Size2 icon_size;
+
+	if (!p_right_icon.is_valid()) {
+		return icon_size;
+	}
+
+	ExpandMode mode = p_get_min ? EXPAND_MODE_FIT_TO_TEXT : expand_mode;
```

Comment:
Why this condition? It's different from original behavior.

https://github.com/user-attachments/assets/1cf83ee7-59dc-4608-a47a-771464c73683

Old:

https://github.com/user-attachments/assets/38e4d538-65bf-4f23-80d0-cfa9fa2cd83c

The icon never leaves the LineEdit.

Reply:
I want the LineEdit to be scaled down as small as possible. That's why it's there. If this changes the default behavior, it should be removed.